cluster-autoscaler/cloudprovider/builder/builder_all.go

```diff
@@ -121,8 +121,9 @@ func buildCloudProvider(opts config.AutoscalingOptions, do cloudprovider.NodeGro
 		return ovhcloud.BuildOVHcloud(opts, do, rl)
 	case cloudprovider.HetznerProviderName:
 		return hetzner.BuildHetzner(opts, do, rl)
-	case cloudprovider.PacketProviderName:
-		return packet.BuildPacket(opts, do, rl)
+	case cloudprovider.PacketProviderName,
```

Comment:
super nit: Can you keep both names in the same line?

Reply:
Done